libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -19,17 +19,24 @@ export function getPackageDeps(packagePath: string = process.cwd(), excludedPath
   };
   const processOutputBlocks: string[] = stdout.split(PROCESS_OUTPUT_DELIMITER);
 
+  // A line is expected to look like:
+  // 100644 blob 3451bccdc831cb43d7a70ed8e628dcf9c7f888c8    src/typings/tsd.d.ts
+  // 160000 commit c5880bf5b0c6c1f2e2c43c95beeb8f0a808e8bac  web-build-tools
+  const gitRegex: RegExp = /([0-9]{6})\s(blob|commit)\s([a-f0-9]{40})\s*(.*)/;
+
```

Comment:
It's not really possible... The current test cases have a big problem where they are actually invoking the Git command on a local folder, which makes it difficult to represent a submodule.

I would be more in favor of doing a refactor on this code to make it be more modular and testable.

What do you think about that?